libraries/postgres/test/database_test.js

```diff
@@ -94,14 +94,14 @@ helper.dbSuite(path.basename(__filename), function() {
             return 'got ' || x;
           end`,
         },
-        add_from_numbers_named: {
+        sub_from_numbers_named: {
```

Comment:
why change this to subtraction?

Reply:
Because addition is commutative so the test would otherwise still pass if the order were reversed (!)

Reply:
ah! makes sense